### PR TITLE
LPS-119456 Replace lfr-asset-item and lfr-asset-folder with Clay 3 card-page classes

### PR DESCRIPTION
**Pull request by pat270**
https://issues.liferay.com/browse/LPS-119456

This replaces `entry-card lfr-asset-item` with Clay 3 `card-page-item card-page-item-asset`,  `entry-card lfr-asset-folder` with Clay 3 `card-page-item card-page-item-directory`, and `display-style-icon list-unstyled row` with Clay 3 `card-page card-page-equal-height`. Modules that are affected:

- asset
- blogs
- depot-web
- document-library
- fragment-web
- item-selector
- web content (journal-web)
- layout admin web
- layout page template admin web
- layout set proptotype web
- mobile device rules
- password policies
- roles admin web

`card-page` is a Clay CSS component that helps layout cards on a page similar to those trading card protectors. It functions exactly like Bootstrap 4's grid.

`card-page-equal-height` forces vertical cards on the same row to be equal heights. This doesn't force all cards inside `card-page` to be exactly the same height.

`card-page-item-asset` functions like Bootstrap's `col-{breakpoint}-#` class except all the sizes and breakpoints are built into the class. This should be used for vertical cards.

`card-page-item-directory` same as above. This should be used for horizontal cards.

/cc @jbalsas

### CODE REVIEW

File: modules/apps/asset/asset-browser-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -136,7 +136,7 @@
 						<c:when test='<%= Objects.equals(assetBrowserDisplayContext.getDisplayStyle(), "icon") %>'>
 
 							<%
-							row.setCssClass("entry-card lfr-asset-item");
+							row.setCssClass("card-page-item card-page-item-asset");
```
Comment (jbalsas) [inline modules/apps/asset/asset-browser-web/src/main/resources/META-INF/resources/view.jsp:139]: Are these classes always expected in a card-view?

Comment (jbalsas) [reply modules/apps/asset/asset-browser-web/src/main/resources/META-INF/resources/view.jsp:139]: Question is... could we add them by default [here](https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/search_iterator/lexicon/icon.jsp#L98) so we don't have to add them manually elsewhere?

Comment (pat270) [reply modules/apps/asset/asset-browser-web/src/main/resources/META-INF/resources/view.jsp:139]: Yes we could. We could probably do something similar to https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/search_iterator/lexicon/icon.jsp#L22-L24 where we output by default unless there are classes passed in?

**Comment by pat270** [comment]
ci:stop

**Comment by pat270** [comment]
@jbalsas I added `card-page-item card-page-item-asset` to `liferay-ui:search-iterator` icon view. If there is no `rowCssClass` it will output those classes by default. If classes are passed in `row.setCssClass()` the classes `card-page-item card-page-item-asset` need to be declared. I think this gives the most flexibility since a lot of GS seems to rely on this taglib. It also is consistent with the way `searchResultsCssClass` works.

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
Nice!! LGTM! Let's get this going!

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**